changelog/unreleased/issue-233

```diff
@@ -0,0 +1,31 @@
+Enhancement: Add negative patterns for include/exclude
+
+If a pattern is suffixed by an exclamation mark and match a file that
```

Comment:
It's probably "prefixed", not "suffixed"?

Reply:
Is changed.